src/controllers/AdminController.java

```diff
@@ -46,92 +45,24 @@ public AdminController(Context context, AdminData adminData) {
     @Override
     public LinkedHashMap<String, Command> AllCommands() {
         LinkedHashMap<String, Command> commands = new LinkedHashMap<>();
-        commands.put("create admin", CreateAdmin());
-        commands.put("create secretary", CreateSecretary());
-        commands.put("create doctor", CreateDoctor());
-        commands.put("create patient", CreatePatient());
         commands.put("change clinic info", changeClinicInformation());
         commands.put("change user password", changeUserPassword());
-        commands.put("delete user", deleteUser());
         commands.put("delete self", deleteSelf());
-
+        commands.put("load user management system", LoadUserManagement());
```

Comment:
The ordering of the commands here is a bit odd. I would suggest starting with the commands that relate to the responsibility of the admin, such as managing users, changing clinic info and then delete self.